app/views/courses/TeacherClassView.coffee

```diff
@@ -253,6 +255,15 @@ module.exports = class TeacherClassView extends RootView
       progressData
       classStats: @calculateClassStats()
     }
+  
+  getCourseAssessmentPairs: () ->
+    # TODO: Associate with the course somehow so courses helper can be used
```

Comment:
Not sure what this TODO means

Reply:
I did the thing it describes; removed the comment now.